platform-coop-toolkit/pcc-framework
# 10517 project post type

## Description

Add project post type

## Steps to test

1. Update local copy of plugin to use this branch
1. In the WordPress admin, go to Page > Add New.
Change Page Attributes > Template to "Project Page".
1. Add a project title and main content.
1. Save the page.

**Expected behavior:** A page can be saved with a project page/post type. The page will be a basic page. Styling is handled in https://github.com/platform-coop-toolkit/pcc/pull/235.
